src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.Xaml/Handler/Breakpoints/ValidateBreakableRangeHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.ExternalAccess.Xaml;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Xaml.Handler;
+
+[ExportXamlStatelessLspService(typeof(ValidateBreakableRangeHandler)), Shared]
+[XamlMethod(LSP.VSInternalMethods.TextDocumentValidateBreakableRangeName)]
+internal sealed class ValidateBreakableRangeHandler : XamlRequestHandlerBase<LSP.VSInternalValidateBreakableRangeParams, LSP.Range?>
```

Comment:
This looks much better!  If the export attribute and handler base were added as external access - would we be able to move even these shim implementations to XAML as well?

For example what we did for some TS things - https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/AbstractVSTypeScriptRequestHandler.cs

And what David did for Razor - https://github.com/dotnet/roslyn/pull/70819/files#diff-0a98c6205f0f7d2c1939afa543a8fef8e6e0d3385ceaed3cf5802cb54eccc0a2

Reply:
I'll give that a on Monday. Otherwise, I believe I have addressed all feedback on this PR.

Reply:
That worked, thank you @dibarbet! I was able to completely remove the LanguageServer.Xaml project.